gbfs.md

```diff
@@ -276,7 +277,7 @@ The `gbfs.json` discovery file SHOULD represent a single system or geographic ar
 
 Field Name | REQUIRED | Type | Defines
 ---|---|---|---
-`feeds` | Yes | Array | An array of all of the feeds that are published by this auto-discovery file. Each element in the array is an object with the keys below.
+`feeds` | Yes | Array<Object> | An array of all of the feeds that are published by this auto-discovery file.
```

Comment:
When the draft PR becomes a Pull Request, let's please replace `<` and `>` with `&lt;` and `&gt;`.

```suggestion
`feeds` | Yes | Array&lt;Object&gt; | An array of all of the feeds that are published by this auto-discovery file.
```

Reply:
Fixed by https://github.com/MobilityData/gbfs/pull/576/commits/0d45ea25dcaaca351cd474a01cf787652655f726